db-setup-core/src/main/java/com/ableneo/liferay/portal/setup/core/SetupArticles.java

```diff
@@ -4,8 +4,7 @@
  * #%L
  * Liferay Portal DB Setup core
  * %%
- * Original work Copyright (C) 2016 - 2018 mimacom ag
- * Modified work Copyright (C) 2018 - 2020 ableneo, s. r. o.
+ * Copyright (C) 2016 - 2019 ableneo s. r. o.
```

Comment:
Hi jsgomezp, thank you for the pull request, please leave the copyright intact.

Reply:
Reverted those lines. Didn't notice they changed.